tmail-backend/apps/memory/pom.xml

```diff
@@ -81,6 +81,10 @@
             <groupId>ch.qos.logback.contrib</groupId>
             <artifactId>logback-json-classic</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Should I have two sub module maven project?
- blobid-list-memory
- blobid-list-cassandra

Reply:
You did forgot to commit `blobid-list/pom.xml?`

Reply:
BTW I am not sure this makes sense for Memory as the problem is specific to Swift S3 APIs....

Reply:
I will remove all memory implement.

Reply:
Sorry, I have some problem with gitignore config :D, I will put pom.xml in the next commit